web/blog/tenzir-node-v4.27/index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Tenzir Node v4.27: Amazon MSK IAM Integration"
+slug: tenzir-node-v4.27
+authors: [lava]
+date: 2025-01-29
+tags: [release, node]
+comments: true
+---
+
+Tenzir Node v4.27 enhances the charting capabilities and integrates with IAM for
+authenticating to Amazon MSK.
+
+![Tenzir Node v4.27](tenzir-node-v4.27.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.27.0
+
+<!-- truncate -->
+
+## AWS IAM Authentication for MSK
+
```

Comment:
Probably a good idea to start with a short introduction of MSK here, ie. what is it and why do people want to use it? And most importantly, why was it not possible to use it with Tenzir before, ie. what new thing are we introducing with this release?